pixl_dcmd/tests/test_main.py

```diff
@@ -194,7 +194,9 @@ def ids_for_parameterised_test(val: pathlib.Path) -> str:
 
 
 @pytest.mark.parametrize(
-    ("yaml_file"), PROJECT_CONFIGS_DIR.glob("*.yaml"), ids=ids_for_parameterised_test
+    ("yaml_file"),
+    PROJECT_CONFIGS_DIR.glob("test-*.yaml"),
```

Comment:
no longer testing anonymisation using the non-test project configs. If we want to test all project configs, we need to add:
```yaml
allowed_manufacturers:
  - regex: "^company"
    exclude_series_numbers: []
```
to all configs, unless there's a nicer way to do it?

Reply:
I think for the existing projects we can just allow all manufacturers, as that's what was implicitly done

Reply:
ah yeah that's fair, and for DESPIAD shall we limit to known manufacturers? And yeah we have a [separate test](https://github.com/SAFEHR-data/PIXL/blob/4afca8acd700f91e7b135da2ec71b394c4e859b3/pixl_core/tests/project_config/test_project_config.py#L91-L97) to check configs are valid

Reply:
I've allowed all manufacturers for the existing configs, and added them back to the `test_anonymise_and_validate_dicom` tests